ios/Classes/CameraPreview/CameraPreview.m

```diff
@@ -102,7 +103,7 @@ - (void)dealloc {
 }
 
 /// Set camera preview size
-- (void)setCameraPresset:(CGSize)currentPreviewSize {
+- (void)setCameraPreviewPresset:(CGSize)currentPreviewSize {
```

Comment:
Why is this renamed to `setCameraPreviewPresset`? It sets the video output preset, not the preview preset.

```suggestion
- (void)setCameraVideoPreset:(CGSize)requestedSize {
```

Reply:
Because it sets the preview preset, not the video output preset: `[_captureSession setSessionPreset:presetSelected];` is the preview.
I assume you're confused because `[CameraQualities selectVideoCapturePresset:currentPreviewSize session:_captureSession];` is used, that's just bad naming by the library, inside the implementation of that method nothing is specific to video.